tla-io/src/main/scala/at/forsyte/apalache/io/ConfigManager.scala

```diff
@@ -4,39 +4,23 @@ import pureconfig._
 import pureconfig.generic.auto._
 import java.io.{File}
 import java.nio.file.{Path, Files, Paths}
+import com.typesafe.config.ConfigValueFactory
+import com.typesafe.config.ConfigObject
 
-object Converters {
+// Provides implicit conversions used when deserializing into configurable values.
+private object Converters {
+  import pureconfig.ConvertHelpers._
 
   private def expandedFilePath(s: String): Path = {
     Paths.get(if (s.startsWith("~")) s.replaceFirst("~", System.getProperty("user.home")) else s)
```

Comment:
Shall we check, whether a string is starting with `"~" + File.separator`. Otherwise, we would misinterpret strings like `~user1`, which have a different meaning in shell.

Reply:
Sorry. Had this configured for auto merge once approved.

yeah, the expansion is not strictly correct. But that’s not actually introduced in this PR: it’s just a continuation of the initial code from the outputManager.

I’ll address this in a follow up tho.